src/player.cpp

```diff
@@ -10112,7 +10106,7 @@ std::vector<Creature *> player::get_targetable_creatures( const int range ) cons
     return g->get_creatures_if( [this, range]( const Creature & critter ) -> bool {
         return this != &critter && pos() != critter.pos() && // TODO: get rid of fake npcs (pos() check)
         round( rl_dist_exact( pos(), critter.pos() ) ) <= range &&
-        ( sees( critter ) || sees_with_infrared( critter ) );
+        ( sees( critter ) || sees_with_infrared( critter ) || sees_with_specials( critter ) );
```

Comment:
I'm not sure "targetable" is appropriate here, does this let you aim and fire guns at targets you can detect with sonar?

Reply:
Yes it does, just tested it on Giant worm, you can shoot at it but the shot goes through it and doesn't connect